dotnet-client-libraries/YamlDotNet/App/Models/Person.cs

```diff
@@ -0,0 +1,10 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace App.Models;
+
+public class Person
+{
+    [Required] public string Name { get; set; }
+
```

Comment:
Since you defined the other classes with the properties defined without blank lines in between, can you remove this blank line too?

Reply:
@grantwinney Done